stripe2qbo/stripe/models.py

```diff
@@ -32,7 +32,162 @@ class Invoice(BaseModel):
     created: int  # timestamp
     due_date: Optional[int] = None
     amount_due: int  # cents
-    currency: Literal["usd", "cad"]
+    currency: Literal[
```

Comment:
thanks for getting started on this! 

I think many of the Stripe models have a currency field, you'll want to update those as well. Define this as a type and re-use it.

We'll also need to add the currencies to the QBO models...

And finally, we'll have to test these! Take a look at `conftest.py` for how the currencies are currently parametrized.

Note: to test this properly, you'll have to make sure your sandbox QBO account has multicurrency enabled.

Something like: settings => account => advances => multicurrency